map/map_tf_generator/test/test_uniform_random.cpp

```diff
@@ -0,0 +1,38 @@
+// Copyright 2020 Tier IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "map_tf_generator/uniform_random.hpp"
+
+#include <gmock/gmock.h>
+
+using testing::AllOf;
+using testing::Each;
+using testing::Ge;
+using testing::Lt;
+
+TEST(UniformRandom, UniformRandom)
+{
+  {
+    const std::vector<int> random = UniformRandom(4, 0);
+    ASSERT_EQ(random.size(), static_cast<long unsigned int>(0));
+  }
+
+  {
```

Comment:
Could you add some comments on what you are testing here?

Reply:
Resolved in https://github.com/autowarefoundation/autoware.universe/pull/890/commits/b7a1945909997e7817df0400f418ca2d2daaf8a3